### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by andreia**
Fix some typos

### CODE REVIEW

**Comment by andreia** [comment]
> Thanks a lot

Thanks for this awesome library! :)
Anything else I should do in this PR before merge?

**Review by aduh95 — APPROVED** [review]
Thanks, good catches

**Comment by Murderlon** [comment]
Seems we have some linter errors still

File: docs/uppy-core.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import UppyCdnExample from '/src/components/UppyCdnExample';
 
 Uppy can be an uploader and an interface with a lot of features. Features can be
 added incrementally with plugins, but Uppy can be as bare bones as you want it
-to be. Therefor we build Uppy’s heart, `@uppy/core`, as a standalone
+to be. So we build Uppy’s heart, `@uppy/core`, as a standalone
 orchestrator. It acts as a state manager, event emitter, and restrictions
 handler.
```
Comment (aduh95) [inline docs/uppy-core.mdx:17]:
```suggestion
to be. So we build Uppy’s heart, `@uppy/core`, as a standalone orchestrator. It
acts as a state manager, event emitter, and restrictions handler.
```

**Pull request merged**